### ISSUE TITLE
A11y_Azure Notebooks_Toolbar_AI4D: Incorrect control as "menu bar" is set for the toolbar.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/fluentui#12433]
**Repro Steps:​​**
1.Launch the url :https://developer.microsoft.com/en-us/fabric#/controls/web/commandbar
2.Launch accessibility insight for desktop and hover over the menu item "New" under "CommandBar with overflowing menu items"

**Actual:​**
Hover over the menu item "new" and select the filter option that comes at top right corner of the box.
Then click on the link "view results in the ui automation tree" and scroll the horizontal scroll bar to 
menu bar and each control in it as menu item .

**Expected:​**
The control should be set as tool bar and the controls in it as buttons.
Same should be announced by the screen readers.

**User Impact:​**
Screen reader users will not get the right information.

**Attachment for reference:**
![image](https://user-images.githubusercontent.com/24580570/77640426-60473d80-6f80-11ea-867f-9ee5804c2adc.png)

**Comment by aneeshack4** [comment]
Closing as dupe of #12353 - see this [link](https://github.com/microsoft/fluentui/issues/12353#issuecomment-601918646) for explanation.
